samples/net/sockets/websocket_client/src/main.c

```diff
@@ -109,6 +114,11 @@ static int setup_socket(sa_family_t family, const char *server, int port,
 				LOG_ERR("Failed to set %s TLS_HOSTNAME "
 					"option (%d)", family_str, -errno);
 				ret = -errno;
+
+				close(*sock);
+				*sock = -1;
+
+				return ret;
```

Comment:
same

Reply:
cal return yes, but close() was not called later stages. So I closed and returned.

Reply:
the close is fine yes. my comments relate to the return ret only

Reply:
Modified a bit.